libzeth/core/bits.hpp

```diff
@@ -35,7 +35,7 @@ template<size_t numBits> class bits : public std::array<bool, numBits>
 
     /// Construct from initializer-list.
     // cppcheck-suppress noExplicitConstructor
-    template<typename... boolList> bits(const boolList &...bits);
+    template<typename... boolList> bits(const boolList &... bits);
```

Comment:
Hum... odd, running `./scripts/format` locally seems to add this space which seems to break the format running on the CI. Need to have a look into this.

Reply:
Worth checking the version of clang-format. (I'm using version 11).